packages/node-opentelemetry/src/sentry.ts

```diff
@@ -157,28 +157,50 @@ const startOtelSpanFromSentryEvent = (event: Event, hint: EventHint) => {
   }
 };
 
-const registerBeforeSendEvent = (event: Event, hint: EventHint) => {
-  hdx('Received event at beforeSendEvent hook');
-  if (isSentryEventAnException(event)) {
-    startOtelSpanFromSentryEvent(event, hint);
+const registerEventProcessor = (event: Event, hint: EventHint) => {
+  try {
+    hdx('Received event');
+    if (isSentryEventAnException(event)) {
+      startOtelSpanFromSentryEvent(event, hint);
+    }
+  } catch (e) {
+    hdx(`Error processing event: ${e}`);
   }
+  // WARNING: always return the event
+  return event;
 };
 
 export const initSDK = async () => {
   try {
     const Sentry = await import('@sentry/node');
-    if (!Sentry.isInitialized()) {
```

Comment:
no longer needed? is this now redundant with just checking if the client exists?

Reply:
yes, we are no longer blocked by this since we use the global event processor, which is the public API